### DISCUSSION TITLE
Downloads slow to a crawl after a while on Big Sur

### DISCUSSION THREAD

**Discussion by datawright** [iterate-ch/cyberduck#14955] [Q&A]
I’ve been having an ongoing problem with CyberDuck on Big Sur where it initially downloads files with no problems, it at some point starts slowing down. Sometimes it slows to a crawl (< 300 KB/sec) and the progress indicator will eventually show that it has downloaded all bytes of the file but keeps going, showing more bytes downloaded than exist in the file and it never stops. Restarting CyberDuck doesn’t help. The only solution is to restart the computer and then it works fine for a while.

As an example of the slowness, I started my computer in the morning and uploaded a couple of 40-50 MB files. Tried downloading an 8.6 GB file and got the following transfer speed (I have a max download speed from my cable provider of around 6 MB/sec). It initially showed around 1.5 MB/sec and then settled down to around 900 KB/sec:

![image](https://github.com/iterate-ch/cyberduck/assets/58000697/39907c7b-4f32-4ec9-9813-56b92e9879cf)

Stopped the download, restarted computer and tried again. It initially showed around 6 MB/sec and then settled down to around 1.6 MB/sec:

![image](https://github.com/iterate-ch/cyberduck/assets/58000697/6b8ba630-8613-447b-bdc8-5ec3e491b9a7)

I'm currently using Version 8.5.4 (39083).

**Comment by tatze96** [comment]
Do you use FTP(S) as the protocol? If so, try if deactivating the segmented downloads within the Cyberduck preferences _Transfers_ tab improves the behavior.

**Reply by datawright** [reply]
The protocol I'm using is SFTP. But I'm turning off the segmented downloads anyway to see if it helps.
